config/locales/it.yml

```diff
@@ -0,0 +1,741 @@
+      hat: "Cappello:"
+      hatplaceholder: "Membro del progetto XYZ"
+      link: "Link:"
+      linkplaceholder: "membro@progetto.org, o URL della pagina dell'azienda"
+      comment: "Commento :"
+      commentplaceholder: "Sarà mostrato solo alla fine del processo di moderazione"
+      requesthatbutton: "Richiedi un cappello"
+    create_request:
+      submittedhatrequest: "Richiesta di cappello inviata con successo."
+    index:
+      title: "Cappello"
+      description: "Un cappello è un modo formale, verificato di postare un commento a nome di un progetto, di un'organizzazione o di un'azienda. Ogni utente può avere più cappelli, uno dei quali può essere scelto per essere indossato quando pubblica un commento o invia un messaggio privato."
+      request: "Richiedi un cappello"
+      user: "Utente"
+      hat: "Cappello"
+      link: "Link"
+    reject_request:
+      rejectedhatrequest: "Domanda di cappello rifiutata."
+    requests_index:
+      title: "Cappelli richiesti"
+      nohatrequests: "Nessuna domanda di cappello."
+      user: "Nome utente :"
+      hat: "Cappello:"
+      link: "Link :"
+      comment: "Commento :"
+      approve: "Approva la richiesta del cappello"
+      reason: "Ragione :"
+      reject: "Rifiutare la richiesta del cappello"
```

Comment:
Rifiutare -> Rifiuta, per coerenza con Approva due righe sopra

Reply:
La forma in francese di "approvare" è all'infinito, è meglio adoperare un altro tempo?

Reply:
secondo me l'imperativo ha più senso su un bottone

Reply:
Rifiuta ci andrebbe meglio in un bottone

Reply:
> secondo me l'imperativo ha più senso su un bottone

Dunque declino tutti gli infiniti contigui come suggerito da @riggraz ?

Reply:
secondo me sì